frame/composable-traits/src/currency.rs

```diff
@@ -46,10 +25,29 @@ where
 
 /// Creates a new asset, compatible with [`MultiCurrency`](https://docs.rs/orml-traits/0.4.0/orml_traits/currency/trait.MultiCurrency.html).
 /// The implementor should ensure that a new `CurrencyId` is created and collisions are avoided.
+/// Is about Local assets representations. These may differ remotely.
 pub trait CurrencyFactory<CurrencyId> {
 	fn create() -> Result<CurrencyId, DispatchError>;
 }
 
+pub trait LocalAssets<MayBeAssetId> {
```

Comment:
Can this get a doc comment, explaining what LocalAssets are?

Reply:
https://github.com/ComposableFi/composable/blob/dz/currency/rfcs/0003-currency.md#what-is-currency